### PR TITLE
chore: add side panel title component

### PR DESCRIPTION
**Pull request by FabPARKJAESUNG**
# Overview
add side panel title component

### CODE REVIEW

File: web/src/beta/components/SidePanelTitle/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+// TabButton.tsx
+import type { FC } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+// 以下のようなPropsでお願いします
```
Comment (atnagata) [inline web/src/beta/components/SidePanelTitle/index.tsx]: Please delete this comment.

File: web/src/beta/components/SidePanelTitle/index.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  border-top-right-radius: 4px;
+`;
+
+const StyledText = styled.text`
+  height: 16px;
+
+  font-family: "Noto Sans";
+  font-style: normal;
+  font-weight: 500;
+  font-size: ${({ fontSize }) => (fontSize ? fontSize : "16px")};
```
Comment (atnagata) [inline web/src/beta/components/SidePanelTitle/index.tsx:44]: Where was the requirement for variable font size?

**Review by atnagata — CHANGES_REQUESTED** [review]
Please correct some of the program source.

File: web/src/beta/components/SidePanelTitle/index.tsx (hunk 2 above)
Comment (FabPARKJAESUNG) [reply web/src/beta/components/SidePanelTitle/index.tsx:44]: variable embedded in text.

**Pull request closed without merging**